sphinx_doc/howtos/configuration/configuring-the-controller.rst

```diff
@@ -0,0 +1,318 @@
+.. _configuring_the_controller:
+
+Configuring The Controller
+##########################
+
+Controller Server
+=================
+
+The controller server responds to FollowPath action requests. It runs a loop
+that continuously computes new command velocities until the goal is reached,
+cancelled or an error occurs. It computes command velocities by using the
+services of a controller plugin. These parameters are set on the
+controller_server node
+
+Parameters
+**********
+
+:controller_frequency:
+  +----------------+----------------------------------------------------------------------------+
+  |**type**        | double                                                                     |
+  +----------------+----------------------------------------------------------------------------+
+  |**default**     |  20.0                                                                      |
+  +----------------+----------------------------------------------------------------------------+
+  |**units**       |  Hz                                                                        |
+  +----------------+----------------------------------------------------------------------------+
+  |**description** |  This parameter controls how often a new command velocity is               |
+  |                |  computed. It also controls how often the controller checks if the goal is |
```

Comment:
> This parameter controls how often a new command velocity is computed.

computed and published? since in each spin publishing takes some CPU time too.